### PR TITLE
Integration tests

### LINKED ISSUE #194: [Feature] Standalone Helm Chart for Etcd Druid
**Feature (What you would like to be added):**
Would be great if we would have a standalone helm chart for druid which can be used outside Gardener.

**Motivation (Why is this needed?):**
Using druid instead of the etcd-operator and utilizing the backup and restore capabilities.

**Approach/Hint to the implement solution (optional):**
Cool thing would be to have a chart in the `charts` repository and released inside this github project via https://github.com/helm/chart-releaser-action in the repositories `gh-pages` branch.

### PR DESCRIPTION
**Pull request by shreyas-s-rao**
Signed-off-by: Shreyas Rao <shreyas.sriganesh.rao@sap.com>

**How to categorize this PR?**
/area testing
/area quality

**What this PR does / why we need it**:
This PR adds cluster-based integration tests for druid.

**Which issue(s) this PR fixes**:
Fixes #194 

**Special notes for your reviewer**:

**Release note**:

```other developer
Add integration tests.
```

### CODE REVIEW

**Comment by shreyas-s-rao** [comment]
@amshuman-kr @abdasgupta PR is ready for review. I will be adding TM support in a separate PR, once https://github.com/gardener/etcd-backup-restore/pull/348 is tested and merged, since the TM integration aspect is similar for both repos.

**Comment by afritzler** [comment]
Any update on this PR? Would be great to have the helm chart for druid.

File: charts/druid/templates/druid-clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
```
Comment (timuthy) [inline charts/druid/templates/druid-clusterrole.yaml:1]: Can you please also add permission for `PodDisruptionBudget`? (https://github.com/gardener/etcd-druid/pull/250)

File: charts/druid/values.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+image: eu.gcr.io/gardener-project/gardener/etcd-druid
+version: v0.6.0-dev-a47e74fd1756951720f8f309cfa34060c677c021
```
Comment (timuthy) [inline charts/druid/values.yaml:2]: Can you remove this default value? I don't think we should have a dev version in Helm chart we offer.

File: charts/druid/values.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+replicas: 0
```
Comment (timuthy) [inline charts/druid/values.yaml:3]:
```suggestion
replicas: 1
```

**Review by timuthy — CHANGES_REQUESTED** [review]
Thanks @shreyas-s-rao. As discussed offline, can you please remove the TM/pipeline changes? I will file a follow up PR.

File: charts/druid/templates/druid-clusterrole.yaml (hunk 1 above)
Comment (timuthy) [reply charts/druid/templates/druid-clusterrole.yaml:1]: In addition, Druid needs the following permissions:
- list/watch/get all `Lease`s
- get/list/watch/create/patch/update `Job`s
- get/list/watch/update/patch `etcdcopybackupstasks`
- get/create/update/patch `etcdcopybackupstasks/status`

**Comment by timuthy** [comment]
As discussed, this PR is replaced by #296.

/close

**Pull request closed without merging**